antarest/study/storage/variantstudy/db/dbmodel.py

```diff
@@ -0,0 +1,83 @@
+import uuid
+from dataclasses import dataclass
+from enum import Enum
+
+from sqlalchemy.orm import relationship  # type: ignore
+
+from antarest.core.persistence import Base
+from sqlalchemy import Column, String, ForeignKey, DateTime, Table, Integer  # type: ignore
+
+from antarest.study.model import (
+    Study,
+)
+
+
+@dataclass
+class VariantStudySnapshot(Base):  # type: ignore
+    """
+    Variant Study Snapshot based entity implementation.
+    """
+
+    __tablename__ = "variant_study_snapshot"
+
+    id = Column(
+        String(36),
+        ForeignKey("variantstudy.id"),
+        primary_key=True,
+    )
+    created_at = Column(DateTime)
+    path = Column(String(255))
```

Comment:
for information : this could be set to <default_workspace>/<study_id>/snapshots/<snapshot_id> when created via the generator